modules/ztp-ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,204 @@
+// TELCODOCS-213 4.8 Using Assisted Installer to Install & Manage ocp Clusters on Bare Metal
+//
+//
+//
+
+[id=ztp-ai-install-ocp-clusters-on-bare-metal_{context}"]
+
+.Enabling Assisted Installer Service on Bare Metal
+
+The Assisted Installer Service (AIS) deploys {product-title} clusters. Red Hat Advanced Cluster Management (RHACM) 2.3+ ships with AIS, which is deployed when the MultiClusterHub Operator is enabled on the RHACM hub cluster.
+
+RHACM 2.3+ only supports *Single Node OpenShift (SNO)* cluster deployments. A SNO cluster runs OpenShift Container Platform (ocp) on top of a single BMO node. When the host is provisoned, the single node cluster acts as both supervisor and worker node.
+
+.Enable assisted installer service
+
+.Prerequisites
+
+. OpenShift Container Platform 4.8+\{product-version}
+. https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.2/html/install/installing#installing-red-hat-advanced-cluster-management-from-the-operatorhub[RHACM 2.3+ with MultiClusterHub resource created]
+. `PersistentVolume` CRs for database and file system storage
+. Install `openshift-install` [cli tool] for the desired version of OCP
+
+.Next Steps
+
+. Modify `HiveConfig` to enable the feature gate for Assisted Installer
++
+[source,bash]
+----
+ oc patch hiveconfig hive --type merge -p '{"spec":{"targetNamespace":"hive","logLevel":"debug","featureGates":{"custom":{"enabled":["AlphaAgentInstallStrategy"]},"featureSet":"Custom"}}}'
+----
+
+. Modify `provisioning` resource to allow the Bare Metal Operator to watch all namespaces
++
+[source,bash]
+----
+ oc patch provisioning provisioning-configuration --type merge -p '{"spec":{"watchAllNamespaces": true }}'
+----
+
+. Create an `AgentServiceConfig` custom resource to install the Assisted Installer Service's controller
++
+[source,bash]
+----
+ cat <<EOF | oc create -f -
```

Comment:
I don't think we should use these kind of cat <<EOF conveniences in the docs, but I could be wrong, my preference would be to have a seperate [source,yaml] block with just the YAML, and then a seperate command to apply or create the resource.